src/Administration/Resources/app/administration/src/app/component/media/sw-media-compact-upload-v2/sw-media-compact-upload-v2.html.twig

```diff
@@ -129,7 +129,7 @@
             <form class="sw-media-upload-v2__form" ref="fileForm">
                 <input class="sw-media-upload-v2__file-input"
                        type="file"
-                       id="files"
+                       :id="createUuid"
```

Comment:
Is this ID used somewhere? When not, why we have that 😅

Reply:
wouldn't this be a breaking change if it's no longer `files`? or even remove it?